HISTORY.md

```diff
@@ -3,6 +3,8 @@
 ### Unreleased
 ### New Features
 * When reading from option file/string/map, customized comparators and/or merge operators can be filled according to object registry.
+* Introduce Periodic Compaction for Level style compaction. Files are re-compacted periodically and put in the same level.
```

Comment:
Is there anything to prevent the logic from being applied to universal too?

Reply:
There is nothing preventing this from working with universal. But I don't want to claim that it is supported with Universal too without further testing. At this point, I haven't tested it with Universal yet (will do so in the future).